deploy-manage/upgrade/deployment-or-cluster/reading-indices-from-older-elasticsearch-versions.md

```diff
@@ -3,50 +3,54 @@ mapped_pages:
   - https://www.elastic.co/guide/en/elasticsearch/reference/current/archive-indices.html
 ---
 
-# Reading indices from older Elasticsearch versions [archive-indices]
+# Reading indices from older {{es}} versions [archive-indices]
 
-{{es}} has full query and write support for indices created in the previous major version. If you have indices created in {{es}} versions 5 or 6, you can now use the archive functionality to import them into newer {{es}} versions as well.
+{{es}} has full query and write support for indices created in the previous major version. If you have indices created in {{es}} versions 5, 6, or 7, you can now use the archive functionality to import them into newer {{es}} versions as well.
 
 The archive functionality provides slower read-only access to older {{es}} data, for compliance or regulatory reasons, the occasional lookback or investigation, or to rehydrate parts of it. Access to the data is expected to be infrequent, and can therefore happen with limited performance and query capabilities.
 
-For this, {{es}} has the ability to access older snapshot repositories (going back to version 5). The legacy indices in the [snapshot repository](../../tools/snapshot-and-restore.md) can either be [restored](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-snapshot-restore), or can be directly accessed via [searchable snapshots](../../tools/snapshot-and-restore/searchable-snapshots.md) so that the archived data won’t even need to fully reside on local disks for access.
+For this, {{es}} can access older snapshot repositories going back to version 5. The legacy indices in the [snapshot repository](../../tools/snapshot-and-restore.md) can either be [restored](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-snapshot-restore) or directly accessed through [searchable snapshots](../../tools/snapshot-and-restore/searchable-snapshots.md) so that the archived data won’t need to fully reside on local disks for access.
 
 
 ## Supported field types [archive-indices-supported-field-types]
 
-Old mappings are imported as much "as-is" as possible into {{es}} 8, but only provide regular query capabilities on a select subset of fields:
+Old mappings are imported as much as possible into {{es}} 9, but only provide regular query capabilities on a select subset of fields:
```

Comment:
I think this can be read in a misleading way (a misreading of "as much as possible" could potentially be seen as potentially data maybe?). What we're trying to say here is that we try to support old mapping types as close as possible but might not be able to support some functionality. So there's no guarantee that the support is 100%, as explained in detail later.